admin_manual/configuration_files/federated_cloud_sharing_configuration.rst

```diff
@@ -2,11 +2,8 @@
 Configuring Federation Sharing
 ==============================
 
-Federated Cloud Sharing is now managed by the Federation app (9.0+), and is 
-now called Federation sharing. When you enable the Federation app you can 
-easily and securely link file shares between Nextcloud servers, in effect 
```

Comment:
I'd prefer to keep the info about having the `federation` app enabled for this feature